src/amo/components/Routes/index.js

```diff
@@ -46,6 +46,11 @@ const Routes = ({ _config = config }: Props = {}) => (
       path="/:lang/:application/addon/:addonSlug/reviews/"
       component={AddonReviewList}
     />
+    <Route
+      exact
+      path="/:lang/:application/addon/:addonSlug/reviews/:reviewId"
```

Comment:
Doesn't this need to come before the route above it since that one matches `/:lang/:application/addon/:addonSlug/reviews/`?

Reply:
I assume not, because it's working fine for me, but I'll move it anyway.

Reply:
Maybe the `exact` in the props is causing it to work?